### DISCUSSION TITLE
如何重新加载 jsMind 配置

### DISCUSSION THREAD

**Discussion by iamjichao** [hizzgdev/jsmind#600] [General]
```js
const options = {
  container: 'jsmind-container',
  editable: true,
  theme: 'default',
  mode: 'side',
};
const jm = new jsMind(options);
```
如果editable是一个动态的值，通过一个开关控制是否可编辑，如何在editable发生改变时reload画布？

**Comment by hizzgdev** [comment]
启用编辑 : 使用 jm.enable_edit() 方法可启用对当前思维导图的编辑功能。
禁止编辑 : 使用 jm.disable_edit() 方法可禁止对当前思维导图进行编辑。

这两个方法可以用来控制是否 editable，并且不需要 reload 画布。

**Reply by iamjichao** [reply]
好的，editable 可以通过专属 API 控制，其他的配置属性有没有通用的办法 reload 呢？

**Reply by hizzgdev** [reply]
其他的配置属性是指哪个呢？可以先看看文档，有些可能有，有些可能没有：

- https://github.com/hizzgdev/jsmind/blob/master/docs/zh/2.options.md
- https://github.com/hizzgdev/jsmind/blob/master/docs/zh/3.operation.md

**Reply by iamjichao** [reply]
是否可以提供一个通用的方法，能够修改options中的任意属性？

**Reply by hizzgdev** [reply]
目前还没有这个打算，不如先介绍一下你的需求呢？
